org.blender.Blender.json

```diff
@@ -202,6 +202,30 @@
                 "/share"
             ]
         },
+        {
+            "name": "opensubdiv",
+            "buildsystem": "cmake-ninja",
+            "builddir": true,
+            "config-opts": [
+                "-DGLEW_LOCATION=/app",
+                "-DNO_EXAMPLES=1",
+                "-DNO_TUTORIALS=1",
+                "-DNO_PTEX=1",
+                "-DNO_DOC=1"
+            ],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/PixarAnimationStudios/OpenSubdiv.git",
+                    "branch": "v3_3_3"
```

Comment:
For better or worse, we use the tarballs for other modules. Can you do that here as well please?